source/Nevermore/CommandExecutor.cs

```diff
@@ -182,17 +180,15 @@ public async Task<DbDataReader> ExecuteReaderAsync(CancellationToken cancellatio
             }
         }
 
-        public async Task<T[]> ReadResultsAsync<T>(Func<DbDataReader, T> mapper)
+        public async Task<T[]> ReadResultsAsync<T>(Func<DbDataReader, T> mapper, CancellationToken cancellationToken = default)
```

Comment:
This is a breaking change to the public contract, either Major version bump or create an overload

Reply:
@droyad I don't think it's possible for these to be an overload because it's an optional param?

RIder indicates that the overload with the optional cancellation token would be hidden

Reply:
CommandExecutor is an internal class, so ignoring this here